sift/packages/python3-pytsk3.sls

```diff
@@ -1,18 +1,13 @@
-{% set package_name = salt['grains.filter_by']({
```

Comment:
We haven't officially stopped supporting bionic yet, any reason you removed this? I believe this is still going to be necessary for bionic support?

Reply:
Referenced in comments, but this package is named identically in both distros now.